charts/kubemq/v0.1.0/questions.yml

```diff
@@ -0,0 +1,63 @@
+labels:
+  io.cattle.role: project
```

Comment:
Can you please add the partner label, thanks
```
  io.rancher.certified: partner
```

Reply:
O.K. Done.
Thanks

Reply:
Made a new pull request: Added KubeMQ message broker. #329